assets/scss/modules/_buttons.scss

```diff
@@ -46,4 +46,45 @@
     }
   }
 }
-@include dp-button();
\ No newline at end of file
+@include dp-button();
+
+.button-loading {
+  position: relative;
+  opacity: 0.8;
+  color: transparent;
+  min-width: 140px;
+  text-shadow: none;
+  text-indent: -9999em;
+
+  &:before {
+    content: '';
+    display: inline-block;
+    position: absolute;
+    background: transparent;
+    border: 2px solid #fff;
```

Comment:
I think we have a variable for `#fff`

Reply:
Please see https://github.com/FromDoppler/Doppler-UI-System/blob/master/assets/scss/core/_colors.scss#L26 `$dp-color-white`

Reply:
Yes